scripts/jenkins/ardana/ansible/templates/input-model-servers.yml

```diff
@@ -10,7 +10,7 @@
 
     - id: ccn-0001
       ip-addr:  {{ deployer_mgmt_ip }}
-      role: LITE-CONTROLLER-ROLE
+      role: CONTROLLER-ROLE
```

Comment:
I guess this requires a change to the deployerincloud-lite in "ardana-input-models"? Is there a patch for that already?

Reply:
hmm, you mean not having different role names between the -lite and non-lite model? I think that would make sense.

Reply:
Well, if you just change it here things will fail for the -lite model, as the roles are referenced in a few other files of the model.

And IMO just aligning them for the deployerincloud-* models makes sense.

Reply:
https://gerrit.suse.provo.cloud/2862

Reply:
Does this naming effect other models or are we considering here to only ever deploy the -lite and the -non-lite models?

Reply:
@nicolasbock we need to have that discussion with someone that has knowledge on the design of input models

Reply:
Ok, sounds good.